lang/en.lyaml

```diff
@@ -39,6 +39,7 @@ en:
           prompts:
             replaceOverrideFile: "Replace existing account override file?"
           errors:
+            globalConfigNotFound: "A global config file could not be found at {{ globalConfigPath }}, which is required for account overrides. Please run {{ authCommand }} to create one."
```

Comment:
Copy suggestion. We could shorten this to: `This command is only compatible with our new global config. Run {{ authCommand }} to get started.`